lib/reporting/mfa_report.rb

```diff
@@ -0,0 +1,218 @@
+# frozen_string_literal: true
+
+require 'csv'
+begin
+  require 'reporting/cloudwatch_client'
+  require 'reporting/cloudwatch_query_quoting'
+  require 'reporting/command_line_options'
+rescue LoadError => e
+  warn 'could not load paths, try running with "bundle exec rails runner"'
+  raise e
+end
+
+module Reporting
+  class MfaReport
+    include Reporting::CloudwatchQueryQuoting
+
+    attr_reader :issuers, :time_range
+
+    module Events
+      MULTI_FACTOR_AUTH = 'Multi-Factor Authentication'
+
+      def self.all_events
+        constants.map { |c| const_get(c) }
+      end
+    end
```

Comment:
I get that it matches the structure of other reports, but since there's just one event (and likely not to change?) maybe we just make it `EVENT` singular constant and just use that?

Reply:
updated in https://github.com/18F/identity-idp/pull/10076/commits/690ebf9fa0667b9e101e9de60df90a53a1eefda0